WordPress/src/main/java/org/wordpress/android/util/EncryptionUtils.java

```diff
@@ -0,0 +1,121 @@
+package org.wordpress.android.util;
+
+import android.content.Context;
+import android.os.Environment;
+import android.util.Base64;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.libsodium.jni.NaCl;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class EncryptionUtils {
+    static final int BOX_SEALBYTES = NaCl.sodium().crypto_box_sealbytes();
```

Comment:
These class constants are a nice tool for keeping things readable!

Reply:
:D Thank you!